src/main/resources/jenkins/plugins/extracolumns/ConfigureProjectColumn/config.jelly

```diff
@@ -23,11 +23,11 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
     <f:block>
         <tr>
             <td width="50">
-                <img src="${imagesURL}/48x48/setting.png" title="${%Configure project icon}" alt="${%Configure project icon}" border="0"/>
+                <l:icon class="icon-gear icon-xlg" title="${%Configure project icon}" alt="${%Configure project icon}" />
```

Comment:
"icon-gear" should be replaced by "icon-setting".

Reply:
I've chosen the gear here to align with the default icon used when configuring jobs.

Reply:
Fair point. I think the gear icon is used to often across the Jenkins UI, but I agree that it should be consistent.